src/main/java/com/candyshop/islodycze/model/Order.java

```diff
@@ -20,18 +22,25 @@
 public class Order {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long ordersId;
+    private Long orderId;
 
     private String totalCost;
 
     private LocalDateTime orderDate;
 
-    @Enumerated(value=EnumType.STRING)
+    @Enumerated(value = EnumType.STRING)
     private OrderStatus orderStatus;
 
     @ManyToOne
-    private User userId;
+    @JoinColumn(name = "user_id_fk")
+    private User userIdFk;
 
     @ManyToMany
-    private Set<ProductOrder> productOrder;
+    @JoinTable(
+            name = "product_orders_orders",
+            joinColumns = @JoinColumn(name = "orders_id"),
+            inverseJoinColumns = @JoinColumn(name = "product_order_id")
+    )
+    @ToString.Exclude
+    private Set<ProductOrder> productOrderFk;
```

Comment:
I think, `Fk` is redundant, but it may stay

Reply:
It is necessary to keep this variable name, because by the default name for relation would be names of variables separated by _.